google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolTest.java

```diff
@@ -1545,6 +1561,35 @@ public void run() {
     assertThat(impl.executePartitionedUpdate(statement)).isEqualTo(1L);
   }
 
+  @Test
+  public void testSessionMetrics() {
+    options =
+        SessionPoolOptions.newBuilder()
+            .setMinSessions(1)
+            .setMaxSessions(3)
+            .setMaxIdleSessions(0)
+            .build();
+    FakeClock clock = new FakeClock();
+    clock.currentTimeMillis = System.currentTimeMillis();
+    FakeMetricRegistry metricRegistry = new FakeMetricRegistry();
+    List<LabelValue> labelValues =
+        Arrays.asList(
+            LabelValue.create("database1"),
+            LabelValue.create("instance1"),
+            LabelValue.create("1.0.0"));
+
+    pool = createPool(clock, metricRegistry, labelValues);
+    runMaintainanceLoop(clock, pool, pool.poolMaintainer.numClosureCycles);
+
+    MetricsRecord record = metricRegistry.pollRecord();
+    assertThat(record.metrics).containsEntry(MetricRegistryConstants.MAX_IN_USE_SESSIONS, 0L);
+    assertThat(record.metrics).containsEntry(MetricRegistryConstants.IN_USE_SESSIONS, 0L);
```

Comment:
I'm not sure what's normal for OpenCensus metrics tests but I was wondering if it makes sense to do something with a session so that the above two metrics have non-zero values.

Reply:
Good point, I used `setupMockSessionCreation` and `get/close` session to get some metrics values. Done in 1525a7d9a1eff5b6d2c7a5573c0c597c7a201c2b PTAL

Reply:
LGTM thanks for updating the test.